src/Chapter2.hs

```diff
@@ -501,7 +517,9 @@ True
 >>> isThird42 [42, 42, 0, 42]
 False
 -}
-isThird42 = error "isThird42: Not implemented!"
+isThird42 :: [Int] -> Bool
+isThird42 [_,_,42,_] = True
```

Comment:
This pattern matching is for 4-elements-only list with the `42` on the third position as it is written in `[]`.
If you want to patent match on __any-sized__ list with 42, you need to use `( : )` construction to pattern match on:

 ```suggestion
isThird42 (_:_:42:_) = True
```